packages/common-utils/src/clickhouse.ts

```diff
@@ -346,7 +346,7 @@ export class ClickhouseClient {
     } else if (isNode) {
       const { createClient } = await import('@clickhouse/client');
       const _client = createClient({
-        host: this.host,
+        url: this.host,
```

Comment:
just curious about this change...related to the PR or a drive-by fix?

Reply:
its not a fix. it should work either way. saw the api deprecation warning (https://github.com/ClickHouse/clickhouse-js/blob/1ebdd39203730bb99fad4c88eac35d9a5e96b34a/CHANGELOG.md?plain=1#L258)